generators/element/index.js

```diff
@@ -0,0 +1,394 @@
+const Generator = require("yeoman-generator");
+
+const _ = require("lodash");
+
+const mkdirp = require("mkdirp");
+const fs = require("fs");
+const process = require("process");
+
+const util = require("util");
+const chalk = require("chalk");
+const yosay = require("yosay");
+
+const packageJson = require("../../package.json");
+
+// Check for a config file to look for defaults
+let config = {};
+if (fs.existsSync("../project.conf.json")) {
+  config = require("../../project.conf.json");
+}
+
+let isPfelement = false;
+let demoTemplatePath;
+let readmePath;
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+      super(args, opts);
+
+      // this.option("type", {
```

Comment:
Should we just remove this code?

Reply:
I would like to be able to create standalone components still.

Reply:
You still can, this portion is setting a default value for type instead of prompting for it (line 48 below) and I'm migrating this default setting to the config file instead of being set here.  I didn't want to nix it entirely yet though in case we wanted to bring it back.

Reply:
I do have to create standalone components from time to time but @mwcz wants to abstract the build tools so the standalone components get the benefit of the build tools updates.

Reply:
Right, this in no way prevents you from creating a standalone. See line 48 -- you can be prompted for it or set it as your default in your config file.